From fae15362db46570db4e26867f966c61fab7bcca7 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Thu, 29 Sep 2016 16:31:39 -0700
Subject: [PATCH] CNS-161 retry backoff for changefeed should use a finite
 timeout CNS-162 updater cueball agent should use higher maximum socket cap

---
 updater.js | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/updater.js b/updater.js
index fd615b7..6ff0b6a 100644
--- a/updater.js
+++ b/updater.js
@@ -46,7 +46,7 @@ var log = bunyan.createLogger({name: 'cns',
 var agent = new cueball.HttpAgent({
 	resolvers: [conf.binder_domain],
 	spares: 2,
-	maximum: 4,
+	maximum: 10,
 	recovery: {
 		default: {
 			timeout: 2000,
@@ -97,8 +97,8 @@ var cfOpts = {
 		subResources: conf.changefeed_opts.subResources
 	},
 	backoff: {
-		maxTimeout: Infinity,
-		minTimeout: 10,
+		maxTimeout: 10000,
+		minTimeout: 2000,
 		retries: Infinity
 	}
 };
-- 
2.21.0

